Rocket.Chat/Managers/DraftMessageManager.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DraftMessageManager.swift
+//  Rocket.Chat
+//
+//  Created by Filipe Alvarenga on 06/11/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+struct DraftMessageManager {
```

Comment:
Documentation this class and it's methods would be also very helpful in future development. It's not required, but you can see how we do for other classes:

https://github.com/RocketChat/Rocket.Chat.iOS/blob/ce241399f8bcfdb712468c6df763c5696883c4e6/Rocket.Chat/Managers/ServerManager.swift#L40-L50

Reply:
done 👍